src/selic-entries.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { selicEntries } from "./selic-entries";
+
+describe("selicEntries", () => {
+    const desiredSelicEntrie = [
+        {
+            "data": '08/03/2024',
+            "valor": '0.041957',
+        }
+    ];
+
+    test("Successful request for a given period", async () => {
+        expect(await selicEntries("08/03/2024","08/03/2024")).toMatchObject(desiredSelicEntrie);
```

Comment:
It's better to make the expectations more explicit.

```suggestion
        expect(await selicEntries("08/03/2024","08/03/2024")).toEqual({
            "data": '08/03/2024',
            "valor": '0.041957',
        });
```